cfme/services/myservice.py

```diff
@@ -1,86 +1,158 @@
-    """
+    @View.nested
+    class myservice(Accordion):  # noqa
+        ACCORDION_NAME = "Services"
 
-    def __init__(self, service_name, vm_name=None, appliance=None):
-        self.service_name = service_name
-        self.vm_name = vm_name
-        Navigatable.__init__(self, appliance=appliance)
+        tree = ManageIQTree()
 
-    def get_detail(self, properties=None, icon_href=False):
-        """ Gets details from the details infoblock
+    configuration = Dropdown('Configuration')
+    policy_btn = Dropdown('Policy')
+    lifecycle_btn = Dropdown('Lifecycle')
+    download_choice = Dropdown('Download')
 
-        Args:
-            *properties: An InfoBlock title, followed by the Key name e.g. "Relationships", "Images"
-            icon_href: Boolean indicating to return icon_href instead of text
-        Returns: A string representing the contents of the InfoBlock's value.
-        """
-        navigate_to(self, 'Details')
-        if icon_href:
-            return InfoBlock.icon_href(*properties)
-        else:
-            return InfoBlock.text(*properties)
+
+class ServiceRetirementForm(MyServicesView):
+    title = Text('#explorer_title_text')
+
+    retirement_date = Calendar('retirementDate')
+    retirement_warning = BootstrapSelect('retirement_warn')
+
+
+class ServiceEditForm(MyServicesView):
+    title = Text('#explorer_title_text')
+
+    name = Input(name='name')
+    description = Input(name='description')
+
+
+class SetOwnershipForm(MyServicesView):
+    title = Text('#explorer_title_text')
+
+    select_owner = BootstrapSelect('user_name')
+    select_group = BootstrapSelect('group_name')
+
+
+class EditTagsForm(MyServicesView):
+    title = Text('#explorer_title_text')
+
+    select_tag = BootstrapSelect('tag_cat')
+    select_value = BootstrapSelect('tag_add')
+
+
+class MyServiceDetailView(MyServicesView):
+    title = Text("#explorer_title_text")
+
+    properties_detail = SummaryTable(title='Properties')
+    lifecycle_detail = SummaryTable(title='Lifecycle')
+    relationships_detail = SummaryTable(title='Relationships')
+    vm_detail = SummaryTable(title='Totals for Service VMs ')
+    smart_mgmnt_detail = SummaryTable(title='Smart Management')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.myservice.is_opened and
+            self.title.text == 'Service "{}"'.format(self.context['object'].name)
+        )
+
+
+class EditMyServiceView(ServiceEditForm):
+    title = Text("#explorer_title_text")
+
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+    cancel_button = Button('Cancel')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.myservice.is_opened and
+            self.title.text == 'Editing Service "{}"'.format(self.context['object'].name)
+        )
+
+
+class SetOwnershipView(SetOwnershipForm):
+    title = Text("#explorer_title_text")
+
```

Comment:
There are two fields in SetownershipForm 
select_owner = BootstrapSelect('user_name')
select_group = BootstrapSelect('group_name') 

And save button here